### PR TITLE
gsagh-239 reaction force diagrams

### PR DESCRIPTION
**Pull request by DominikaLos**

### CODE REVIEW

File: GsaGH/Components/5_Results/ReactionForceDiagrams.cs [hunk 1]
```diff
@@ -0,0 +1,508 @@
...
+using GsaGH.Parameters;
+using OasysGH;
+using OasysGH.Components;
+using OasysGH.Units;
+using OasysGH.Units.Helpers;
+using OasysUnits;
+using OasysUnits.Units;
+using Rhino.Geometry;
+
+namespace GsaGH.Components._5_Results
```
Comment (tlmnrnhrdt) [inline GsaGH/Components/5_Results/ReactionForceDiagrams.cs:22]: Can we move this to namespace `GsaGH.Components` please?

File: GsaGH/Components/5_Results/ReactionForceDiagrams.cs [hunk 2]
```diff
@@ -0,0 +1,508 @@
...
+{
+  /// <summary>
+  /// Component to display GSA node result contours
+  /// </summary>
+  public class ReactionForceDiagrams : GH_OasysDropDownComponent
+  {
+    #region core
+    protected override void RegisterInputParams(GH_InputParamManager pManager)
```
Comment (tlmnrnhrdt) [inline GsaGH/Components/5_Results/ReactionForceDiagrams.cs:30]: Sorry, I meant putting those after the constructor (before all the other methods..)

File: GsaGH/Components/5_Results/ReactionForceDiagrams.cs [hunk 3]
```diff
@@ -0,0 +1,508 @@
...
+    private bool _undefinedModelLengthUnit = false;
+    private ForceUnit _forceUnit = DefaultUnits.ForceUnit;
+    private MomentUnit _momentUnit = DefaultUnits.MomentUnit;
+    private DisplayValue _selectedDisplayValue = DisplayValue.ResXYZ;
+    #endregion
+
+    #endregion
+
+    #region public methods
+    public ReactionForceDiagrams() : base("Reaction Force Diagrams",
```
Comment (tlmnrnhrdt) [inline GsaGH/Components/5_Results/ReactionForceDiagrams.cs:159]: that´s technically a constructor and not a method no?

File: GsaGH/Components/5_Results/ReactionForceDiagrams.cs [hunk 4]
```diff
@@ -0,0 +1,508 @@
...
+      this._momentUnit = (MomentUnit)UnitsHelper.Parse(typeof(MomentUnit), reader.GetString("moment"));
+      return base.Read(reader);
+    }
+    #endregion
+    
+    public override void DrawViewportWires(IGH_PreviewArgs args)
+    {
+      base.DrawViewportWires(args);
+
+      _reactionForceVectors.ForEach(force =>
```
Comment (tlmnrnhrdt) [inline GsaGH/Components/5_Results/ReactionForceDiagrams.cs:222]: should we use `this` everywhere?

File: GsaGH/Components/5_Results/ReactionForceDiagrams.cs [hunk 5]
```diff
@@ -0,0 +1,509 @@
...
+using OasysGH.Units;
+using OasysGH.Units.Helpers;
+using OasysUnits;
+using OasysUnits.Units;
+using Rhino.Geometry;
+
+namespace GsaGH.Components
+{
+  /// <summary>
+  /// Component to display GSA node result contours
```
Comment (tlmnrnhrdt) [inline GsaGH/Components/5_Results/ReactionForceDiagrams.cs:25]: this should say something about reaction forces no?

**Review by tlmnrnhrdt — CHANGES_REQUESTED** [review]

**Review by kpne — APPROVED** [review]

**Pull request merged**